styles/globals.scss

```diff
@@ -160,11 +160,7 @@ $xxl: 1400px;
     @apply font-450 tracking-copy;
   }
 
-  .full-width-bg {
-    @apply ml-[50%] w-screen -translate-x-1/2;
-
-    &__inner {
-      @apply mx-auto max-w-site px-6 lg:px-16;
-    }
+  .wrapper {
+    @apply mx-auto max-w-site px-6 lg:px-16;
   }
```

Comment:
so with `dir=rtl` alone (9b958dc750bf3f51a610e35f8dfdd52803ca24d7), we have a very odd bug that i couldn't figure out, something to do with the `full-width-bg` utility's absolute positioning, 2d transforms, and margin together:
<img width="1624" alt="Screen Shot 2022-07-18 at 20 58 15" src="https://user-images.githubusercontent.com/1642599/179648004-ce69d211-f1d4-4c4d-abf5-7bab4a451199.png">

at a certain point i just decided i probably over-engineered it originally. unwrapping the default site max-width and using a more traditional `.wrapper` class is more, well, traditional, but seems more reliable in this rtl context since i can't wrap my head around the issue above

Reply:
oo, i reverted to https://github.com/oakstudios/joinmastodon/commit/9b958dc750bf3f51a610e35f8dfdd52803ca24d7 and  I tried the full width class from [here](https://css-tricks.com/full-width-containers-limited-width-parents/#aa-with-known-non-parent-width) and this seems to fix it:
```
  .full-width-bg {
    margin-left: calc(50% - 50vw);
    margin-right: calc(50% - 50vw);

    &__inner {
      @apply mx-auto max-w-site px-6 lg:px-16;
    }
  }
```